cpp/CMakeLists.txt

```diff
@@ -763,6 +763,22 @@ if(HT_DEFAULT_ALLOCATOR)
     set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -DHT_DEFAULT_ALLOCATOR")
 endif(HT_DEFAULT_ALLOCATOR)
 
+# Set a default logging level if none was specified
+set(DEFAULT_LOGGING_LEVEL OFF)
+
+if(NOT LOGGING_LEVEL)
```

Comment:
Is there a way we can just inherit this from the RMM cmake?

Reply:
cudf is currently picking up RMM via a conda package which does not ship any cmake files, so I don't know of a way to do this as RMM is located by libcudf today.

I think we'd either have to add a cmake snippet to the librmm conda package and have libcudf pick up the cmake settings from there, or we'd need to have libcudf pull RMM's source as it does Thrust/CUB today.   However I don't know if avoiding the RMM conda package and pulling RMM source is something we want to do.  @kkraus14 

I suspect the RMM cmake source may also need to be refactored to make picking this up from other projects easier to do.

Reply:
Eventually the plan will be to export the cmake info in the `librmm` conda package so someone can do `find_package(rmm)` and get a nice target that allows inheriting options like these. We're still likely a bit away from that though.